lib/clean-git.nix

```diff
@@ -102,7 +103,7 @@ then
     gitModules = builtins.path { name = "gitmodules"; path = gitModulesStr; };
 
     gitSubmoduleFiles = cleanSourceWith {
-      inherit src;
+      inherit name src;
```

Comment:
I'd have thought you want different names for these, otherwise they're going to look confusingly like the main src. e.g. `name = name + "-gitSubmoduleFiles"`

Reply:
Good idea.